Validation/RecoTrack/src/MTVHistoProducerAlgoForTracker.cc

```diff
@@ -2026,6 +2056,7 @@ void MTVHistoProducerAlgoForTracker::fill_generic_recoTrack_histos(const Histogr
     histograms.h_recozpos[count]->Fill(vertz);
     histograms.h_recodr[count]->Fill(dR);
     histograms.h_recodrj[count]->Fill(dRJet);
+    histograms.chi2_vs_drj[count]->Fill(dRJet, chi2);
```

Comment:
I think the problem in the @mmusich plot can be here: I am filling this plot here (in the recoTrack part), instead of simAssociated_recoTrack (like the chi2_vs_otherVariables), because dR(jet,track) is not available at sim level. If this produce unreasonable results I'll remove it (we still a have "assoc_chi2_vs_drj" for similar purpose).
Marco in which folder/histo_name did you get this picture. Only to be suere we are talking of exactly the same plot.

Reply:
Valerio, I clicked semi-randomly on several folders so that exact plot I am not sure, but e.g. you can look to
` (Top) / Tracking / TrackBuilding / detachedTripletStep_quickAssociatorByHits` or the likes..